### PR TITLE
Volvo beta test day 1,2,3

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 2/02_00_ARC02_1.txt [hunk 1]
```diff
@@ -74,7 +74,7 @@ Undeterred and unexisting, you are dreaming of a story that has already passed.
 [sha:cfe91652b93028e1cc2070298a745a152b90c9e5]{
 -- Page 5, Offset 2609.
 -- <欠けていく月のように|貴方の影を求めるように>。
-<Much as the moon wanes|M u c h   a s   I   y e a r n   f o r   y o u r   s h a d o w>.
+<Much as the moon wanes|M u c h   a s   I   y e a r n   f o r   y o u r   s h a d o w          >.
```
Comment (Volvo-hackangel) [inline script/Arcueid/Day 2/02_00_ARC02_1.txt:77]: This one looks like this btw. Is that ok or should the ruby align with the text below? image

Comment (Volvo-hackangel) [reply script/Arcueid/Day 2/02_00_ARC02_1.txt:77]: Better? image

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 2]
```diff
@@ -167,7 +167,7 @@ They really did have a delicacy like that on the menu, back in the day.
 -- Page 59, Offset 3760.
 -- あまりの高コストで一週間で生産中止になったが、今でも復活を待つ生徒は少なくない。
 They were so expensive that they were discontinued after the first week,
- but a lot of students are still holding out hope they'll make a comeback.
+ but a lot of students are still holding out hope they'd make a comeback.
```
Comment (rschlaikjer) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:170]: > ...holding out hope they **will** make a comeback

original is correct

File: script/Ciel/Day 2/02_01_CIEL02_1_3.txt [hunk 3]
```diff
@@ -848,7 +848,7 @@ Ciel stares at Arihiko in exasperation until he disappears from the courtyard.
 -- 「あいつ、必要最低限のコトはきっちりこなしてますからね。自己の不始末は自分で片付けるヤツだし。先輩も信頼して放任してるんじゃないですか？」
 "That guy's an expert at doing only as much as he has to.
  He's the kind of guy who cleans up his own mess.
- I think you should just trust in him and let him be."
+ I think you should just trust him and let him be."
```
Comment (rschlaikjer) [inline script/Ciel/Day 2/02_01_CIEL02_1_3.txt:851]: Trust in and trust are slightly different, I think trust in is correct here.

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 4]
```diff
@@ -419,7 +419,7 @@ As I watch the two of them out of the corner of my eye, I gaze idly at the scene
 -- 「うわあ、すごいなあ。わたし、一食でも抜いたら動けません。おなかがぺこぺこになって倒れちゃいます。
 "Wow, that's amazing.
  I can't function if I skip even a single meal.
- I get so hungry that I just collapse.
+ I get so hungry that I just might collapse.
```
Comment (rschlaikjer) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:422]: Original more correct to me

File: script/Common/00_00.txt [hunk 5]
```diff
@@ -808,6 +808,7 @@ I don't want to be anywhere riddled with these lines.
 [sha:41ec505aac191030b79a22cbb6798394b30aabb5]{
 -- Page 53, Offset 260. Mods: @n.
 -- ここから逃げ出して、
+// can that second I decide be removed or at least be replaced?
```
Comment (rschlaikjer) [inline script/Common/00_00.txt:811]: Can probably just drop to 
> I decide to make my escape.
> To flee somewhere...

File: script/QA/QA_08.txt [hunk 6]
```diff
@@ -38,7 +38,7 @@ Hello there. Are you feeling well-rested after the%{n}
 [sha:103ec3e13c5f415d3c90e7ce21e229f85de9aab1]{
 -- Page 160, Offset 43055.
 -- 十分にお休みできましたか？
-lack of dead ends in days 6-8?
+lack of dead ends in days six to eight?
```
Comment (rschlaikjer) [inline script/QA/QA_08.txt:41]: six _through_ eight I think

File: script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 7]
```diff
@@ -235,7 +235,7 @@ Freedom... no, egotism would describe it better. These are rights that you can o
 -- Page 322, Offset 1979.
 -- 「君は遠野の子でありながら、それを理解していなかった。秋葉とは似ても似つかぬ三流だ。まあ、一度地に落ちていながら帰ってくる厚顔さだけは、一流と言えなくもないがね」
 "You are a son of the Tohno family, but you didn't understand that.
- While similar to Akiha at first glance, you're really sub-par.
+ While similar to Akiha at first glance, you're really subpar.
  Well, your nerve is up to par at least, seeing as how you came back here after falling so low."
```
Comment (Petrikow) [inline script/Arcueid/Day 1/01_00_ARC01_6_1.txt:239]: I'd probably prefer this construction so the other meaning of "really" doesn't cause a misinterpretation.
>...glance, you're subpar, really.

File: script/Arcueid/Day 1/01_00_ARC01_8.txt [hunk 8]
```diff
@@ -67,7 +67,7 @@ The white walls are reminiscent of castle ramparts.
 [sha:c12b2e20ac73967fe6ae8926d6a9f390dc8f12ed]{
 -- Page 449, Offset 2303.
 -- 温かみのあるオレンジ色の電灯が延々と続く中、メイド服の女の子が歩いている。
-Down this endless corridor of warm orange lights walks a girl in a maid uniform.
+Down this endless corridor of warm orange lights walks a girl in a maid's uniform.
```
Comment (Petrikow) [inline script/Arcueid/Day 1/01_00_ARC01_8.txt:70]: almost everywhere I look "maid uniform" is more common. And by a lot. Even "maid outfit" is more common

File: script/Arcueid/Day 1/01_00_ARC01_8.txt [hunk 9]
```diff
@@ -619,7 +619,7 @@ Now that I think about it, they do look pretty similar...
 -- Page 493, Offset 2410.
 -- 終始笑顔だった着物の少女に比べると、このメイド服の少女はあまりにも無表情なので、二人が姉妹だと結びつかなかったのだ。
 Compared to the girl in the kimono, who was smiling the whole time,
- the girl in the maid uniform is so expressionless that I didn't even make the connection that the two were sisters.
+ the girl in the maid's uniform is so expressionless that I didn't even make the connection that the two were sisters.
```
Comment (Petrikow) [inline script/Arcueid/Day 1/01_00_ARC01_8.txt:622]: see above

File: script/Arcueid/Day 2/02_00_ARC02_1.txt [hunk 10]
```diff
@@ -149,7 +149,7 @@ Where am I?
 [sha:df1fb888cb7c713383877c1cf10bb21693436034]{
 -- Page 10, Offset 2624.
 -- メイド服の少女がおじぎをする。
-The girl in the maid uniform bows.
+The girl in the maid's uniform bows.
```
Comment (Petrikow) [inline script/Arcueid/Day 2/02_00_ARC02_1.txt:152]: see above

File: script/Arcueid/Day 3/03_00_ARC03_3.txt [hunk 11]
```diff
@@ -77,7 +77,7 @@ What magnanimity. I'm at a loss for words.
 [sha:26b0d26648653f5fd8a6ec4b78c544daa75612ef]{
 -- Page 132, Offset 4284.
 -- 「いや、それでも三年と二年じゃ距離が離れすぎです。ＨＲ、遅刻しても知りませんよ」
-"Alright, but the third- and second-year classrooms are pretty far apart.
+"Alright, but the third-year and second-year classrooms are pretty far apart.
  Don't blame me if you end up late for homeroom."
```
Comment (Petrikow) [inline script/Arcueid/Day 3/03_00_ARC03_3.txt:81]: you'd probably say it "third and second-year" if you were talking, so this should stay

maybe the dash on third isn't necessary but that's another question

File: script/Arcueid/Day 1/01_00_ARC01_8.txt (hunk 8 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 1/01_00_ARC01_8.txt:70]: alright sure

File: script/Arcueid/Day 3/03_00_ARC03_3.txt (hunk 11 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 3/03_00_ARC03_3.txt:81]: Im going with the former since I also dont think the hyphen is necessary

**Pull request merged**